explorer/src/components/account/StakeAccountSection.tsx

```diff
@@ -114,12 +139,17 @@ function OverviewCard({
             </td>
           </tr>
         )}
-        {!stakeAccount.meta && (
```

Comment:
I think we could change this to a conditional `showDelegation` that evaluates to true whenever a stake account type is not equal to "delegated" or when a delegation has been fully deactivated. Then, `showDelegation` could be used to show/hide the delegations card and the stake account status

Reply:
Is the `showDelegation` logic backwards? We want to hide the Delegation card when fully deactivated or type !== delegation?